### PR TITLE
added in support for "secure grains" and "secrets" dir

### PR DESCRIPTION
**Pull request by verhulstm**

### CODE REVIEW

**Comment by verhulstm** [comment]
this pr is also related to issue #218

**Comment by verhulstm** [comment]
and related to issue 184

File: rambo/Vagrantfile [hunk 1]
```diff
@@ -101,6 +101,9 @@ custom_code_dirs = [
 Vagrant.configure("2") do |config|
   if get_env_var_rb('SYNC') != 'disabled'
     config.vm.synced_folder '.', "/vagrant", type: get_env_var_rb('SYNC')
+    if get_env_var_rb("SECRETS_PATH")
+      config.vm.synced_folder File.join(get_env_var_rb("CWD"), get_env_var_rb("SECRETS_PATH")), "/vagrant/secrets", type: get_env_var_rb('SYNC')
```
Comment (nixjdm) [inline rambo/Vagrantfile]: On a technical note, I think `get_env_var_rb("SECRETS_PATH")), "/vagrant/secrets"`  could just be added to the list of synced custom dirs above in line 91. That would keep this block simpler.

**Comment by nixjdm** [comment]
Closed by #358, arbitrary syncing and minimal provisioning.

**Pull request closed without merging**